examples/storm-kafka-client-examples/pom.xml

```diff
@@ -73,19 +75,27 @@
             <groupId>org.apache.storm</groupId>
             <artifactId>storm-kafka-client</artifactId>
             <version>${project.version}</version>
+            <!-- You can reduce jar size by uncommenting this and providing the dependencies manually. See the README for details.
             <scope>${provided.scope}</scope>
```

Comment:
Isn't the goal of ${provided.scope} to handle the proper scope according to the profile, e.g. just like it is done with the Intellij profile. I am not quite following why the user has to comment/uncomment the scope configuration.